268 rta all docs (#3844)
timescale/docs: use-timescale/data-retention/about-data-retention.md

# About data retention

> In modern applications, data grows exponentially and often becomes less useful in day-to-day operations as it gets older. 

Reviewer: ```suggestion
In modern applications, data grows exponentially. As data gets older, it often becomes less useful in day-to-day operations. 
```

However, you still need it for analysis. Timescale elegantly solves this problem with
[automated data retention policies][retention-policy]. 

> Once a policy is set up, it deletes raw old data for you on a schedule. 

Reviewer: ```suggestion
Data retention policies delete raw old data for you on a schedule that you define. 
```
